app/views/notebooks/index.rss.builder

```diff
@@ -26,7 +26,7 @@ xml.rss(version: '2.0') do |rss| # rubocop: disable Metrics/BlockLength
       channel.item do |item|
         item.title nb.title
         item.description nb.description
-        item.link "#{request.base_url}#{notebook_path(nb, ref: :rss)}"
```

Comment:
This is the only one of these _path switchouts that was truly necessary. Going to roll back the rest most likely